src/test/java/com/google/javascript/clutz/types_in_zip.d.ts

```diff
@@ -0,0 +1,25 @@
+declare namespace ಠ_ಠ.clutz.types_in_zip {
+  var a : number ;
```

Comment:
can you keep the zip tests more minimal, there is no specific logic in the translation itself when it comes to .zip inputs, so this is just another golden for us to update.

Reply:
Add instructions how to generate the .zip test file.

Reply:
Made tests smaller, I still wanted to have 2 tests in the zip, so I left 2 anyways.
I added some documentation on how to add zip tests in the generated declarations file and in the `getGoldenFile`, should I add something else anywhere else?